en/docs/guides/identity-lifecycles/configure-recaptcha-for-self-registration.md

```diff
@@ -53,7 +51,7 @@ You can either configure the recaptcha for a tenant, or configure it globally.
 
 1.  Navigate to the
     `           <IS_HOME>/repository/conf/deployment.toml          `
-    file and uncomment the following configuration block.
+    file and add the following configurations.
```

Comment:
after the toml configuration steps, better to mention to save the changes and restart the server